modules/product/product.graphql.js

```diff
@@ -0,0 +1,50 @@
+const productsType = `
+type Products {
+_id: ID!
+category: Category!
+subcategory: Category!
+name: [Language]!
+description: [Language]!
+mainMaterial: [Language]!
+innerMaterial: [Language]!
+strapLengthInCm: Int!
+images: [PrimaryImage]!
+colors: [Color]!
+pattern: [Language]
+patternImages: ImageSet
+closure: [Language]!
+closureColor: String
+basePrice: Int!
+options: [ProductOptions]!
+available: Boolean!
+isHotItem: Boolean!
+purchasedCount: Int
+rate: Float
+rateCount: Int
+comments: [Comments]
+}
+`;
+
+const productsInput = `
+input productsInput {
```

Comment:
Please, rename it to `ProductInput`. For the future, all inputs should be named in a format `${ModelName}Input`

Reply:
Renamed